diff --git a/dist/common/client.js b/dist/common/client.js
index d7f55e6f575fbd0420f7b07d748f8099ab0c4091..61de13e6d18000438049dff18b8798fec77b35cd 100644
--- a/dist/common/client.js
+++ b/dist/common/client.js
@@ -1,4 +1,4 @@
-import { parseQueryKey } from './query-key';
+import { parseQueryKey } from './query-key.js';
 export function invalidateQueriesMatchingPredicate(queryClient, predicate) {
     return queryClient.invalidateQueries({
         predicate: ({ queryKey }) => {
diff --git a/dist/vue.js b/dist/vue.js
index 86d50efce143183bc65511e2770ad92394ee2dee..093c44864aaae330028db833e89300a05ad63861 100644
--- a/dist/vue.js
+++ b/dist/vue.js
@@ -3,9 +3,9 @@ import { createInvalidator, createOptimisticUpdater, DEFAULT_QUERY_ENDPOINT, } f
 import { fetcher, makeUrl, marshal } from '@zenstackhq/client-helpers/fetch';
 import { lowerCaseFirst } from '@zenstackhq/common-helpers';
 import { computed, inject, provide, toValue, unref } from 'vue';
-import { getAllQueries, invalidateQueriesMatchingPredicate } from './common/client';
-import { CUSTOM_PROC_ROUTE_NAME } from './common/constants';
-import { getQueryKey } from './common/query-key';
+import { getAllQueries, invalidateQueriesMatchingPredicate } from './common/client.js';
+import { CUSTOM_PROC_ROUTE_NAME } from './common/constants.js';
+import { getQueryKey } from './common/query-key.js';
 export const VueQueryContextKey = 'zenstack-vue-query-context';
 /**
  * Provide context for query settings.
